### ISSUE TITLE
Unicode character removed from string

### ISSUE THREAD

**Issue by vhiairrassary** [scalameta/scalafmt#1033]
Unicode character removed from string after formatting:

- **Version**: `"com.lucidchart" % "sbt-scalafmt" % "1.9"`
- **Integration**: `sbt`
- **Configuration**:
```
maxColumn = 120
runner.optimizer.forceConfigStyleMinArgCount = 1
docstrings = JavaDoc
spaces.inImportCurlyBraces = true
```

## Steps

I ran scalafmt like this:

```bash
sbt scalafmt
```
on the following input
```scala
package dataoncloud

object Main extends App {
  val color = 31
  val message = "hello"

  println(s"\u001b[1;${color}m${message}\u001b[m")
}
```

## Problem

Scalafmt formats code like this:
```scala
package dataoncloud

object Main extends App {
  val color = 31
  val message = "hello"

  println(s"\u001b[1;${color}m${message}b[m")
}
```

## Expectation

I would like the formatted output to look like this:
```scala
package dataoncloud

object Main extends App {
  val color = 31
  val message = "hello"

  println(s"\u001b[1;${color}m${message}\u001b[m")
}
```

**Comment by olafurpg** [comment]
Thank you for reporting! I am able to reproduce, a minimal example is `s"${1}\u001b"` This is a bug, the unicode character should remain

**Comment by olafurpg** [comment]
This seems fixed in master after #1170, opened #1175 and I'm unable to reproduce the bug

**Comment by vhiairrassary** [comment]
👍
